### ISSUE TITLE
User albums not showing in /photo/albums/view_myalbums/ when user created albums while logged in as a page. No activity points recorded either for items uploaded while logged in as page.

### ISSUE THREAD

**Issue by hamadax** [PHPfox-Official/phpfox-v4-issues#2628]
# Important

Please follow this template!

----------------------------------------

## What's happened?
When a user creates a bunch of albums while logged in as a page, these albums do not show under the user's albums. The only way for the user to see these albums is to find the page they were created under, log in as that page, then locate the album. More interesting also is that no activity points are given to the user if they upload photos or videos while logged in as a page.

## Steps to reproduce:
1. Create a page, login as that page, create photo album, upload photos to that album, login back as user, check your albums... The albums created in your page are not showing there.
2. Check activity points. Nothing recorded.
3. Same issue with activity points not being recorded for added videos when logged in as a page.

## What's expected?
Show all albums created by the user under /photo/albums/view_myalbums/ including albums created while logged in as a page.

## Browsers and Devices tested
(Example: Chrome on iPhone X, Safari on Macbook, Miscrosoft Edge on Windown 10, Firefox on Ubuntu 16.04, ...)

all

## Server information
(Example: CentOS 7, php 7.1 apache)

all

## phpFox version
(Example: phpFox 4.6.0)

4.7.5

## Screenshots
...

**Comment by harrison05** [comment]
Yes, when you logging as page and add new photo Album, the owner of this album is your Page. So you can't find it in **My Album** page and you can't get activity points. 

Regards.

**Comment by hamadax** [comment]
OK, if this is working as designed, please consider updating the language files. It clearly tells the user that they are getting a point for uploading a photo. No exclusions mentioned. Thanks!

**Comment by harrison05** [comment]
When you log in as page and upload photo you can't get activity point, this is normal because in that time users don't act on their account anymore, I don't think it will make they confuse.

Regards.

**Comment by hamadax** [comment]
Yes I totally get what you are stating, but a general good common sense approach to design and usability is NOT to make the end user think what the underlying code does and does not do. Sure, the user is logged in as a page, but that does not mean the user is automatically deprived of a point for contributing content, just because, they logged in as a page - unless you want to tell them first! So if you follow this logic, where are the activity points earned by that page? It's acting as a separate account or entity, unlike groups, where activity points work just fine and there is no functionality to login as a group creator! The group admin get their activity points counted.

What I am trying to say here, is we don't tell a user that they will a get "a point" for uploading "a photo" or "a video", then limit that functionality in pages without warning or any kind of exclusion statement. This is deceptive imo. Note that you are making the assumption that the user already knows that activity points will be turned off when they log in as a page. You only know this as the developer, the user has no clue until they find out and when they do, it either looks deceptive or erroneous.

I looked in the language file and I cannot find a sentence that I can use to tell the user what you are telling me. Please suggest how I can tell the user that their activity points will not count when logged in as a page?

Thanks.

**Comment by harrison05** [comment]
Ok, I understand you. I have a quick solution for this case. 
As you know, user can see total points they will get when adding an item at **Activity Points > How To Earn**
![Screen Shot 2019-04-25 at 09 39 31](https://user-images.githubusercontent.com/41947198/56706134-1174a080-673e-11e9-897e-1e016d368ad3.png)

You can search phrases at **AdminCP > Globalization > Phrases** and update it to explain about this case
**Example:** I search with text "Specify how many points the user will receive when adding a new blog." and see the result:

> <title>Activity points</title><info>Specify how many points the user will receive when adding a new blog.</info>

Then I update text in **<info>** tag (don't need to change text in **<title>** tag) to: 

> <title>Activity points</title><info>Specify how many points the user will receive when adding a new blog (User won't get point if post as page).</info>

Then save and clear cache. Now users will know this exception.

I hope this may help you.

Regards.

**Comment by hamadax** [comment]
I see what you are saying, but it's not ideal. The same phrases are being used in the points transaction view. It will look odd to keep repeating the same disclaimer thousands of times. I think I would rather edit the template of that page. Will get to it eventually. Thanks!

![image](https://user-images.githubusercontent.com/48427415/56901220-a8ef3180-6a65-11e9-8810-1a3dfac1b6be.png)

**Comment by spirogg** [comment]
@hamadax @harrison05 

you can make this easy and add the test under the pages  Activity Points > How To Earn

this shows how many points  added for creating a page>
- then add text under it that say pages are not included in any other points activity for photos, videos, likes, comments etc.

just an Idea so its not redundant under all Activity Points > How To Earn ONLY PAGES SECTION ?

just an Idea 

![pages-explaination](https://user-images.githubusercontent.com/17736688/96965358-9f674e00-14d1-11eb-8a89-d91859efb63d.PNG)
